aries-site/src/examples/templates/wizard/WizardValidationExample.js

```diff
@@ -340,7 +408,7 @@ const CancellationLayer = ({ onSetOpen }) => {
           <Heading color="text-strong" margin="none">
             Cancel
           </Heading>
-          <Text color="text-strong">Action Title</Text>
+          <Text color="text-strong">Wizard Title</Text>
         </Box>
```

Comment:
Is this Box extraneous? Can we get away with a fragment instead?

Reply:
Yes, Fragment is fine.